src/hooks/index.js

```diff
@@ -552,6 +552,13 @@ exports.sendEmail = (context, maildata) => {
 	const userIds = (typeof maildata.userIds === 'string' ? [maildata.userIds] : maildata.userIds) || [];
 	const receipients = [];
 
+	// create email attachments
+	const files = maildata.attachments || [];
+	const attachments = [];
+	files.forEach((element) => {
+		attachments.push({ filename: element.name, content: Buffer.from(element.data) });
```

Comment:
check buffer size that it does not break our receiver postbox max file size limit, in that case throw an error... ask falko for this...